Dockerfile

```diff
@@ -42,26 +57,10 @@ COPY --from=ui /out /app/ui/out
 COPY --from=provider-ui /out /app/provider-ui/out
 COPY --from=wrk2 /wrk2 /app/cmd/wrk2
 COPY --from=wrk2 /wrk2/wrk /usr/local/bin
+COPY --from=seed_content /seed_content /home/appuser/.meshery/seed_content
+COPY --from=layer5/getnighthawk:latest /usr/local/bin/nighthawk_service /app/cmd/
+COPY --from=layer5/getnighthawk:latest /usr/local/bin/nighthawk_output_transform /app/cmd/

```

Comment:
You're a clever man, @DelusionalOptimist. I love the ingenuity here. Also, it probably makes sense for us to publish binary releases on the getnighthawk repo.

Reply:
Gotcha :+1: https://github.com/layer5io/getnighthawk/issues/185